cmd/tempo/main.go

```diff
@@ -283,3 +282,35 @@ type otelErrorHandlerFunc func(error)
 func (f otelErrorHandlerFunc) Handle(err error) {
 	f(err)
 }
+
+func setMutexBlockProfiling(mutexFraction int) {
+	mutexPercent := os.Getenv("PPROF_MUTEX_PROFILING_PERCENT")
```

Comment:
search the codebase for `intFromEnv` perhaps we consolidate this logic?

Reply:
I am going to remove the env vars and add a new command flag since one already exists for mutex.